### ISSUE TITLE
Cursor does not update for selections any more

### ISSUE THREAD

**Issue by rolandlo** [xournalpp/xournalpp#2833]
**Affects versions :**
 - OS: Linux (Ubuntu), but probably all OS'es 
 - X11 
 - libgtk 3.24.23 
 - Version of Xournal++: 1.1.0+dev 
 - Installation method: PPA

**Describe the bug**
When moving the mouse near a selection the cursor does not change to the rotate cursor or (one of the 4) resizing cursors when hovering over it

**To Reproduce**
1. Create a selection (with the rectangle tool, for instance)
2. Hover with the mouse cursor over the rotate button
3. Observe that you don't get a rotate cursor

**Expected behavior**
The cursor should update as it did in earlier versions

**Additional context**
This is a regression of PR #2749. It seems that a motion action was prematurely ignored if there was no input running. This doesn't work with hovering events.

**Comment by personalizedrefrigerator** [comment]
Can confirm. #2749 also seems to have only been a partial fix. I'm opening a PR to revert it.

**Comment by Febbe** [comment]
@personalizedrefrigerator do you've done a new PR fixing both?

**Comment by personalizedrefrigerator** [comment]
> do you've done a new PR fixing both?

No, but (at least for me), I've stopped experiencing automatic pen activation on page crossing. I'm not sure when this was fixed or if a change in my configuration is making me not experience the issue...
